src/components/ImagesAndVideos.tsx

```diff
@@ -0,0 +1,73 @@
+import { Card, Flex } from "antd";
+import { getImage, GatsbyImage } from "gatsby-plugin-image";
+import React from "react";
+import { ParentalLineFrontmatter } from "../component-queries/types";
+import { formatCellLineId } from "../utils";
+const {
+    container,
+    titleSection,
+    mainTitle,
+    subtitle,
+    rightTitle,
+    imageSection,
+    caption,
+    imageContainer,
+} = require("../style/images-and-videos.module.css");
+
+interface ImagesAndVideosProps {
+    images?: any[];
+    cellLineId: string;
+    parentalLine: ParentalLineFrontmatter;
+    videos?: any;
+    geneSymbol: string;
+    snp: string;
+}
+
+const ImagesAndVideos: React.FC<ImagesAndVideosProps> = ({
+    images = [],
+    cellLineId,
+    parentalLine,
+    geneSymbol,
+}) => {
+    const mainImage = images.length > 0 ? images[0] : null;
+    const imageData = mainImage ? getImage(mainImage.image) : null;
+    const fluorescentTag = parentalLine.fluorescent_tag;
+    const parentalGeneSymbol = parentalLine.gene.frontmatter.symbol;
+    const alleicTag = parentalLine.allele_count;
+    const title = (
+        <Flex
+            justify="space-between"
+            style={{ paddingTop: 24 }}
+            className={titleSection}
+        >
+            <div>
+                <h3 id={mainTitle}>{formatCellLineId(cellLineId)}</h3>
+                <span className={subtitle}>
+                    {geneSymbol}-(placeholder) in WTC-{fluorescentTag}-{parentalGeneSymbol}
+                    ({alleicTag}-allelic tag)
+                </span>
```

Comment:
Using the parental line ID to dynamically determine `WTC` will be addressed in a separate PR. (I'm a little confused about the duplicated name -`parental_line` in admin/config.yml and how it maps in graphql)